subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -416,7 +431,7 @@ int aws_fota_mqtt_evt_handler(struct mqtt_client *const client,
 		   (evt->param.suback.message_id == SUBSCRIBE_JOB_ID_UPDATE)) {
 			err = update_job_execution(client, job_id,
 						   AWS_JOBS_IN_PROGRESS,
-						   fota_state,
+						   fota_state, 0,
```

Comment:
Does this mean a new download has started. If so, should stored_offset be reset here for good measure?

Reply:
Yes, and good idea :)